lighthouse-plugin-publisher-ads/audits/viewport-ad-density.js

```diff
@@ -64,22 +63,24 @@ class ViewportAdDensity extends Audit {
       return auditNotApplicable.NoVisibleSlots;
     }
 
-    const adArea = slots.reduce((sum, slot) =>
-      sum + boxViewableArea(slot.clientRect, viewport), 0);
-
-    const viewArea = viewport.innerWidth * viewport.innerHeight;
-
-    if (viewArea <= 0) {
+    if (viewport.innerHeight <= 0) {
```

Comment:
Why not also check for width?

Reply:
We're not measuring density vertically. I'd leave that to lighthouse to verify that